src/PayableServiceProvider.php

```diff
@@ -23,6 +25,21 @@ public function register()
      */
     public function boot()
     {
-        //
+        $this->loadRoutesFrom(__DIR__.'/routes/payment_routes.php');
+
+        if ($this->app->runningInConsole()) {
+
+            $this->commands([
+                PaymentSetupCommand::class,
+            ]);
+
+            $destination = app_path('Http/Controllers/Payment/PaymentCallbackController.php');
+
+            if (!File::exists($destination)) {
+                $this->publishes([
+                    __DIR__.'/Http/Controllers/PaymentCallbackController.php.stub' => $destination,
```

Comment:
**Important could be blocking**

__DIR__ will return the current directory which I think it's out side src directory. if this is true then we need to specify the right beginning of the path